provisioning/ansible/roles/web/tasks/main.yml

```diff
@@ -40,8 +40,8 @@
 - name: Copy SSL configuration
   copy: src=ssl.conf dest=/etc/nginx/conf.d/ssl.conf owner=nginx group=nginx mode=600
 
-- name: Create PHP session directory
-  shell: mkdir /var/lib/php/session creates=/var/lib/php/session
+- name: Creates directory
+  file: path=/var/lib/php/session state=directory owner=nginx group=nginx mode=0775
```

Comment:
Note: permissions on /var/lib/php/session are managed by rpm and are reset by rpm on updates. Moving the session dir solves this:
https://github.com/SURFnet/Stepup-Deploy/blob/develop/roles/app/tasks/main.yml#L75
See also: https://www.pivotaltracker.com/story/show/114071895